src/components/Tag/Tag.css

```diff
@@ -1,5 +1,5 @@
 .tag {
-    width: 150px;{
```

Comment:
you generally dont want to push code not-related to the swipe page. Things like this, you will file it as a bug on trello and let people know.